db/models/flags_v2/assessment_flag.py

```diff
@@ -0,0 +1,57 @@
+from uuid import uuid4
+
+from db import db
+from db.models.flags_v2.flag_update import FlagStatus
+from flask_sqlalchemy import DefaultMeta
+from sqlalchemy import Column
+from sqlalchemy import ForeignKey
+from sqlalchemy import String
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import relationship
+
+BaseModel: DefaultMeta = db.Model
+
+
+class AssessmentFlag(BaseModel):
```

Comment:
might we be able to use a dataclass here with a post init: https://www.infoworld.com/article/3563878/how-to-use-python-dataclasses.html#:~:text=other%20instance%20data%3A